docs/meson.build

```diff
@@ -0,0 +1,8 @@
+if get_option('enable-gtk-doc')
```

Comment:
Does it currently generate and triggers the documentation?

Reply:
Yes, well it generates the same documentation as before (which is only a version.entities file inside the builddir/docs dir). Documentation should be improved definetely, but I believe should be a separate task.